008-infinispan-getting-started/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.quarkus.qe</groupId>
+    <artifactId>beefy-scenarios</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>008-infinispan-getting-started</artifactId>
+
+  <properties>
```

Comment:
hm, https://github.com/quarkusio/quarkus/blob/master/bom/test/pom.xml#L12 could be used

Reply:
what about this @kshpak ?

Reply:
@rsvoboda just to clarify. Do you mean to add the `<dependencyManagement>` section in the child POM with testcontainers BOM import?

Reply:
yeah, that was the plan

but there is no such BOM in https://maven.repository.redhat.com/ga/io/quarkus/ :( aka for product

Lets keep version definition, but bump it to the latest (1.14.3, info from https://mvnrepository.com/artifact/org.testcontainers/testcontainers)

Reply:
Ok, need to change it back :D

Reply:
@rsvoboda Final version ready. Just bumped to the latest version